### DISCUSSION TITLE
Refactoring variable bounds (min/max/fix)

### DISCUSSION THREAD

**Discussion by mihlema** [spine-tools/SpineOpt.jl#431] [Ideas]
During our last call, we talked about possibly refactoring our existing variable bounds. Currently, it is possible to directly fix a variable to a certain value through the `fix_variable_name` parameters. We could extend this functionality, to also allow the user to force a `lower/upper_bound` on variables. This way the user can e.g. conveniently define, how many units of a certain type need to be at least invested in. Implementation would be relatively straightforward, populating our workaround for fixing variables and adding `min/max_variable_name`.
(Related issue #416 )

So far we haven't identified any downsides, except for obviously more parameter definitions.

@Spine-project/spineopt-developers

**Comment by clizbe** [comment]
@mihlema This is a pretty old discussion. Did we implement it? Can we close the discussion?
